src/functions/views/provider.php

```diff
@@ -17,3 +17,69 @@ function tribe_events_tickets_views_v2_is_enabled() {
 
 	return tribe_events_views_v2_is_enabled();
 }
+
+add_action( 'tribe_settings_before_content_tab_event-tickets', 'event_tickets_render_settings_banner' );
+
+/**
+ * Render the Help banner for the Ticket Settings Tab.
+ *
+ * @since TBD
+ *
+ * @return string The help banner HTML content.
+ */
+function event_tickets_render_settings_banner() {
+	$et_resource_links = [
+		[
+			'label' => __( 'Getting Started Guide', 'event-tickets' ),
+			'href'  => 'https://theeventscalendar.com/knowledgebase/guide/event-tickets/',
+		],
+
+		[
+			'label' => __( 'Configuring PayPal for Ticket Purchases', 'event-tickets' ),
+			'href'  => 'https://theeventscalendar.com/knowledgebase/k/configuring-paypal-for-ticket-purchases/',
+		],
+		[
+			'label' => __( 'Configuring Tribe Commerce', 'event-tickets' ),
+			'href'  => 'https://theeventscalendar.com/knowledgebase/k/configuring-tribe-commerce/',
+		],
+		[
+			'label' => __( 'Managing Orders and Attendees', 'event-tickets' ),
+			'href'  => 'https://theeventscalendar.com/knowledgebase/k/tickets-managing-your-orders-and-attendees/',
+		],
+		[
+			'label' => __( 'Event Tickets Manual', 'event-tickets' ),
+			'href'  => 'https://theeventscalendar.com/knowledgebase/product/event-tickets/',
+		],
+	];
+
+	$etp_resource_links = [
+		[
+			'label' => __( 'Tickets & WooCommerce', 'event-tickets' ),
+			'href'  => 'https://theeventscalendar.com/knowledgebase/k/woocommerce-specific-ticket-settings/',
+		],
+
+		[
+			'label' => __( 'Creating Tickets', 'event-tickets' ),
+			'href'  => 'https://theeventscalendar.com/knowledgebase/k/making-tickets/',
+		],
+		[
+			'label' => __( 'Event Tickets and Event Tickets Plus Settings Overview', 'event-tickets' ),
+			'href'  => 'https://theeventscalendar.com/knowledgebase/k/settings-overview-event-tickets-and-event-tickets-plus/',
+		],
+		[
+			'label' => __( 'Event Tickets Plus Manual', 'event-tickets' ),
+			'href'  => 'https://theeventscalendar.com/knowledgebase/product/event-tickets-plus/',
+		],
+	];
+
+	$context = [
+		'etp_enabled'        => class_exists( 'Tribe__Tickets_Plus__Main' ),
+		'et_resource_links'  => $et_resource_links,
+		'etp_resource_links' => $etp_resource_links,
+	];
+
+	/** @var Tribe__Tickets__Admin__Views $admin_views */
+	$admin_views = tribe( 'tickets.admin.views' );
+
+	return $admin_views->template( 'settings/getting-started', $context );
+}
```

Comment:
This doesn't seem like the correct place for this set of function/hook. I would love this to be in a class.